### PR TITLE
Bring our own GUAC helm charts.

### PR DESCRIPTION
**Pull request by bobmcwhirter**
Update DEVELOPING.md for helmstuff.

### CODE REVIEW

File: deploy/k8s/chart/Chart.yaml [hunk 1]
```diff
@@ -7,9 +7,9 @@ version: 0.1.0
 appVersion: "1.16.0"
 
 dependencies:
-- name: guac
-  version: "0.1.x"
-  repository: "https://kusaridev.github.io/helm-charts"
+- name: nats
```
Comment (jcrossley3) [inline deploy/k8s/chart/Chart.yaml:10]: I thought we got rid of nats. What am I misremembering?

File: deploy/k8s/chart/templates/guac/collectsub/030-Deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{{ if .Values.guac.enabled }}
```
Comment (jcrossley3) [inline deploy/k8s/chart/templates/guac/collectsub/030-Deployment.yaml:1]: To me, it's strange to see this condition in a file beneath a directory called "guac". I would rather see this condition in whatever file is referencing this file... I think.

File: deploy/k8s/chart/templates/guac/collectsub/040-Service.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{{ if .Values.guac.enabled }}
```
Comment (jcrossley3) [inline deploy/k8s/chart/templates/guac/collectsub/040-Service.yaml:1]: ibid

File: deploy/k8s/chart/templates/guac/graphql/030-Deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
+{{ if .Values.guac.enabled }}
```
Comment (jcrossley3) [inline deploy/k8s/chart/templates/guac/graphql/030-Deployment.yaml:1]: ibid

File: deploy/k8s/chart/templates/guac/graphql/040-Service.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+{{ if .Values.guac.enabled }}
```
Comment (jcrossley3) [inline deploy/k8s/chart/templates/guac/graphql/040-Service.yaml:1]: ibid

**Review by jcrossley3 — APPROVED** [review]
Without knowing much about helm or guac... I approve... I guess?

File: deploy/k8s/chart/templates/guac/collectsub/040-Service.yaml (hunk 3 above)
Comment (bobmcwhirter) [reply deploy/k8s/chart/templates/guac/collectsub/040-Service.yaml:1]: Ovid

File: deploy/k8s/chart/templates/guac/graphql/030-Deployment.yaml (hunk 4 above)
Comment (bobmcwhirter) [reply deploy/k8s/chart/templates/guac/graphql/030-Deployment.yaml:1]: Ibsen

File: deploy/k8s/chart/templates/guac/graphql/040-Service.yaml (hunk 5 above)
Comment (bobmcwhirter) [reply deploy/k8s/chart/templates/guac/graphql/040-Service.yaml:1]: Bison

File: deploy/k8s/chart/Chart.yaml (hunk 1 above)
Comment (bobmcwhirter) [reply deploy/k8s/chart/Chart.yaml:10]: Nope. At least the Bombastic exporter deployment still indicates a desire to have nats.

We used to get it transitively from the dependency upon the kusari guac charts.

**Pull request merged**